srcpkgs/s6-man-pages/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 's6-man-pages'
 pkgname=s6-man-pages
-version=2.11.0.0.2
+version=2.11.0.0.4
 revision=1
 build_style=gnu-makefile
-hostmakedepends="mdocml"
 short_desc="Ports of the HTML documentation for the s6 supervision suite to mdoc(7)"
 maintainer="mobinmob <mobinmob@disroot.org>"
 license="ISC"
 homepage="https://github.com/flexibeast/s6-man-pages"
 distfiles="https://github.com/flexibeast/s6-man-pages/archive/v${version}.tar.gz"
-checksum=0936736aec214ee0be94ecafa6501003d5b36cd5e7e157a00b9d5caf27cd3682
+checksum=83d51bc0fd688dc48f84f1b80f55c9c9831a6c000902421947eed1eecfc102cd
 
-
-do_install() {
-	vmkdir usr/share/man/man1
+pre_install() {
 	vmkdir usr/share/man/man7
-	make MANPATH="${DESTDIR}/usr/share/man"
+	vmkdir usr/share/man/man8
+}
+
+post_install() {
 	vlicense LICENSE
 }
```

Comment:
Any reason not to do `vlicense` in `pre_install`?

Reply:
Done ;)